copyright

```diff
@@ -46,3 +46,22 @@ files:
  Copyright: bene-dictator (benstaples8068@gmail.com)
 License: CC-BY-SA-4.0
 Comment: Derived from works by Michael Zahniser (mzahniser@gmail.com), Nate Graham (pointedstick@zoho.com), and Tommaso Becca (tommasobecca03@gmail.com).
+
+files:
+ images/outfit/bunrodea fuel bead.png
+ images/outfit/compressed fuel pod.png
+ Copyright: Michael Arsollon (midnightplugins@outlook.com)
+License: CC-BY-SA-4.0
+Comment: Derived from works by Michael Zahniser (under the same license)
```

Comment:
Putting this PR on indefinite hold while I wait for Midnight to get back to me on whether it's okay to use his art here.